client/analytics/report/orders/constants.js

```diff
@@ -0,0 +1,95 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const filters = [
+	{ label: __( 'All Orders', 'wc-admin' ), value: 'all' },
+	{
+		label: __( 'Single Order', 'wc-admin' ),
+		value: 'single',
+		subFilters: [
+			{
+				label: __( 'Single Order', 'wc-admin' ),
+				component: 'Search',
+				value: 'single_order',
+			},
+		],
+	},
+	{ label: __( 'Top Orders by Items Sold', 'wc-admin' ), value: 'top_items' },
+	{ label: __( 'Top Orders by Gross Sales', 'wc-admin' ), value: 'top_sales' },
+	{ label: __( 'Advanced Filters', 'wc-admin' ), value: 'advanced' },
+];
+
+export const filterPaths = {
+	all: [],
+	single: [],
+	single_order: [ 'single' ],
+	top_items: [],
+	top_sales: [],
+	advanced: [],
+};
+
+export const advancedFilters = {
+	matches: [
+		{ value: 'all', label: __( 'All', 'wc-admin' ) },
+		{ value: 'any', label: __( 'Any', 'wc-admin' ) },
+	],
+	filters: {
+		status: {
+			label: __( 'Order Status', 'wc-admin' ),
+			addLabel: __( 'Order Status', 'wc-admin' ),
+			rules: [
+				{ value: 'is', label: __( 'Is', 'wc-admin' ) },
+				{ value: 'is-not', label: __( 'Is Not', 'wc-admin' ) },
+			],
+			input: {
+				component: 'SelectControl',
+				options: [
+					{ value: 'pending', label: __( 'Pending', 'wc-admin' ) },
```

Comment:
Some extensions add custom order stati - TIL that we can grab those via the REST API, but might be easier to get them into `wcSettings` on page load and utilize them here. [Here is the API logic](https://github.com/woocommerce/woocommerce/blob/master/includes/api/v1/class-wc-rest-orders-controller.php#L902-L910) that grabs the status list.

Reply:
Oh for sure I meant to say we can do that in a follow-up PR.